<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>[nutmeg] - alpha v0.4</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="workArea"/>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="listArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export To</string>
     </property>
     <addaction name="actionText_File"/>
     <addaction name="actionPEGASUS"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionImport"/>
    <addaction name="menuExport"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuColor_Scheme">
     <property name="title">
      <string>Color Scheme</string>
     </property>
     <addaction name="actionLight"/>
     <addaction name="actionDark"/>
    </widget>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionOptions"/>
    <addaction name="menuColor_Scheme"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionTutorial"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionUndo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionText_File">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Text File</string>
   </property>
  </action>
  <action name="actionPEGASUS">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>PEGASUS</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionKeybinds">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionTutorial">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Controls</string>
   </property>
  </action>
  <action name="actionGithub">
   <property name="text">
    <string>Github</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New Window</string>
   </property>
  </action>
  <action name="actionLight">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Light</string>
   </property>
  </action>
  <action name="actionDark">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
